identity-service/src/routes/recovery.js

```diff
@@ -56,6 +56,15 @@ module.exports = function (app) {
       return errorResponseBadRequest('Invalid signature provided, no user found')
     }
 
+    await models.UserEvents.update(
```

Comment:
@raymondjacobson how do you feel about putting this after the try catch post email being sent?

Reply:
g00d call!